marklogic-data-hub-central/ui/src/components/modeling/graph-view/relationship-modal/add-edit-relationship.module.scss

```diff
@@ -101,10 +101,31 @@
 }
 .cardBody {
   position: relative;
-  padding: 24px !important;
+  padding: 24px 24px 24px 0 !important;
 
 }
 
+.cardIcon {
+  width: 40px;
+
+  svg {
+    width: 24px;
```

Comment:
Let's create a mixin for the CSS given under svg tag here. That way you can reuse it at multiple places(like at line 123) by using @include. It would make the code cleaner.

Reply:
After fix the alignment it's not needed the mixin